### PR TITLE
Added articles: Material Design and User interface design for the English and Spanish language

### PR DESCRIPTION
**Pull request by FahedHermoza**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

### CODE REVIEW

**Comment by RandellDawson** [comment]
@lynxlynxlynx What do you think about this article and the Spanish translation of it?  Is it good to be merged?

**Comment by lynxlynxlynx** [comment]
Too Spanish for me to review, but it's for the guide anyway.

File: guide/english/android-development/material-design/index.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+The animation in Android became very powerful in each version, nowadays there are many animation effects, with the purpose of providing continuity between different states of UI to create more comprehensible experiences.
+
+
+## Adaptive Design
+The adaptive design changes the UI based on the size of the screen, whether it is on a Nexus 4 (4.7 inches), Pixel XL (5.5 inches), Nexus 6 (6.0 inches) or a Nexus 10 (10.1 inches) phone ).
+
+Breakpoints are points where the UI changes. When finding a breakpoint you should start to change the UI. You can start with the drawable, string, layout, orientation files. For specific adaptations, you can vary the number of columns, heights, and width of margins, space.
+
+# More information
+- [Material Design para Android](https://developer.android.com/design/material/?hl=es-419).
```
Comment (moT01) [inline guide/english/android-development/material-design/index.md]:
```suggestion
- [Material Design para Android](https://developer.android.com/design/material/?hl=es-419)
```

File: guide/english/android-development/material-design/index.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+- [Material Components for Android](https://github.com/material-components/material-components-android).
```
Comment (moT01) [inline guide/english/android-development/material-design/index.md]:
```suggestion
- [Material Components for Android](https://github.com/material-components/material-components-android)
```

File: guide/english/android-development/material-design/index.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+- [Material design specifications](https://material.io/design/introduction/#).
```
Comment (moT01) [inline guide/english/android-development/material-design/index.md]:
```suggestion
- [Material design specifications](https://material.io/design/introduction/#)
```

File: guide/english/android-development/material-design/index.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+- [Material Design](https://www.youtube.com/watch?v=Q8TXgCzxEnw).
```
Comment (moT01) [inline guide/english/android-development/material-design/index.md]:
```suggestion
- [Material Design](https://www.youtube.com/watch?v=Q8TXgCzxEnw)
```

File: guide/english/android-development/material-design/index.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+- [Material Design in the Google I/O App](https://www.youtube.com/watch?v=XOcCOBe8PTc&hl=es-419).
```
Comment (moT01) [inline guide/english/android-development/material-design/index.md]:
```suggestion
- [Material Design in the Google I/O App](https://www.youtube.com/watch?v=XOcCOBe8PTc&hl=es-419)
```

File: guide/english/android-development/material-design/index.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+- [Google I/O 2014 - Material design: Structure and components](https://www.youtube.com/watch?v=dZqzz5lZFvo).
```
Comment (moT01) [inline guide/english/android-development/material-design/index.md]:
```suggestion
- [Google I/O 2014 - Material design: Structure and components](https://www.youtube.com/watch?v=dZqzz5lZFvo)
```

File: guide/english/android-development/material-design/index.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+- [User interfaces using Material Design by Uplabs](https://www.uplabs.com/android).
```
Comment (moT01) [inline guide/english/android-development/material-design/index.md]:
```suggestion
- [User interfaces using Material Design by Uplabs](https://www.uplabs.com/android)
```

File: guide/english/android-development/material-design/index.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+
+# Courses
+- [Material Design for Android Developers](https://www.udacity.com/course/material-design-for-android-developers--ud862).
```
Comment (moT01) [inline guide/english/android-development/material-design/index.md]:
```suggestion
- [Material Design for Android Developers](https://www.udacity.com/course/material-design-for-android-developers--ud862)
```

File: guide/english/android-development/material-design/index.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+- [Codelab](https://codelabs.developers.google.com/codelabs/material-design-style-sp/index.html#0).
```
Comment (moT01) [inline guide/english/android-development/material-design/index.md]:
```suggestion
- [Codelab](https://codelabs.developers.google.com/codelabs/material-design-style-sp/index.html#0)
```

File: guide/english/android-development/user-interface-desing/index.md [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+setContentView(R.layout.main_activity);
+```
+The setContentView method displays a layout, that is, an interface and displays it on the screen. When deploying a layout, an instance of each design element contained in the layout file is created, according to the definition of its attributes. To specify which layout is deployed, we pass to the method the resource Id of that layout.
+```
+Button button = (Button) findViewById(R.id.button);
+```
+To connect an element of the layout file to the Activity we use the findViewById method. This method accepts the resource Id of a widget as an argument and returns a View object.
+
+# Más Información
+- [Layouts](https://developer.android.com/guide/topics/ui/declaring-layout?hl=en-419).
```
Comment (moT01) [inline guide/english/android-development/user-interface-desing/index.md]:
```suggestion
- [Layouts](https://developer.android.com/guide/topics/ui/declaring-layout?hl=en-419)
```

File: guide/english/android-development/user-interface-desing/index.md [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+- [Layout resource](https://developer.android.com/guide/topics/resources/layout-resource?hl=es-419).
```
Comment (moT01) [inline guide/english/android-development/user-interface-desing/index.md]:
```suggestion
- [Layout resource](https://developer.android.com/guide/topics/resources/layout-resource?hl=es-419)
```

File: guide/english/android-development/user-interface-desing/index.md [hunk 12]
```diff
@@ -0,0 +1,71 @@
...
+
+# Cursos
+- [Material Design for Android Developers](https://www.udacity.com/course/material-design-for-android-developers--ud862).
```
Comment (moT01) [inline guide/english/android-development/user-interface-desing/index.md]:
```suggestion
- [Material Design for Android Developers](https://www.udacity.com/course/material-design-for-android-developers--ud862)
```

File: guide/english/android-development/user-interface-desing/index.md [hunk 13]
```diff
@@ -0,0 +1,71 @@
...
+- [CodeLabs](https://codelabs.developers.google.com/codelabs/material-design-style-sp/index.html#0).
```
Comment (moT01) [inline guide/english/android-development/user-interface-desing/index.md]:
```suggestion
- [CodeLabs](https://codelabs.developers.google.com/codelabs/material-design-style-sp/index.html#0)
```

File: guide/spanish/android-development/material-design/index.md [hunk 14]
```diff
@@ -0,0 +1,59 @@
...
+
+La animación en Android se volvió muy poderoso en cada versión, en la actualidad existen muchos efectos de animación, con el propósito de proporcionar continuidad entre diferentes estados de UI para crear experiencias más comprensibles.
+
+## Diseño Adaptativo
+El diseño adaptativo cambia la UI basado en el tamaño de la pantalla, ya sea que se encuentre en un teléfono Nexus 4 (4.7 pulgads), Pixel XL (5.5 pulgadas), Nexus 6 (6.0 pulgadas) o una tablet Nexus 10 (10.1 pulgadas).
+
+Los breakpoints son puntos donde la UI cambia. Al encontrar un breakpoint debe empezar a cambiar la UI. Usted puede empezar por los archivos drawables, string, layout, orientación. Para adaptaciones especificas usted puede variar el numero de columnas, alturas y anchura de márgenes, espacio.
+
+# Más Información
+- [Material Design para Android](https://developer.android.com/design/material/?hl=es-419).
```
Comment (moT01) [inline guide/spanish/android-development/material-design/index.md]:
```suggestion
- [Material Design para Android](https://developer.android.com/design/material/?hl=es-419)
```

File: guide/spanish/android-development/material-design/index.md [hunk 15]
```diff
@@ -0,0 +1,59 @@
...
+- [Material Components for Android](https://github.com/material-components/material-components-android).
```
Comment (moT01) [inline guide/spanish/android-development/material-design/index.md]:
```suggestion
- [Material Components for Android](https://github.com/material-components/material-components-android)
```

File: guide/spanish/android-development/material-design/index.md [hunk 16]
```diff
@@ -0,0 +1,59 @@
...
+- [Material design specifications](https://material.io/design/introduction/#).
```
Comment (moT01) [inline guide/spanish/android-development/material-design/index.md]:
```suggestion
- [Material design specifications](https://material.io/design/introduction/#)
```

File: guide/spanish/android-development/material-design/index.md [hunk 17]
```diff
@@ -0,0 +1,59 @@
...
+- [Material Design](https://www.youtube.com/watch?v=Q8TXgCzxEnw).
```
Comment (moT01) [inline guide/spanish/android-development/material-design/index.md]:
```suggestion
- [Material Design](https://www.youtube.com/watch?v=Q8TXgCzxEnw)
```

File: guide/spanish/android-development/material-design/index.md [hunk 18]
```diff
@@ -0,0 +1,59 @@
...
+- [Material Design in the Google I/O App](https://www.youtube.com/watch?v=XOcCOBe8PTc&hl=es-419)
+- [Google I/O 2014 - Material design: Structure and components](https://www.youtube.com/watch?v=dZqzz5lZFvo).
```
Comment (moT01) [inline guide/spanish/android-development/material-design/index.md]:
```suggestion
- [Google I/O 2014 - Material design: Structure and components](https://www.youtube.com/watch?v=dZqzz5lZFvo)
```

File: guide/spanish/android-development/material-design/index.md [hunk 19]
```diff
@@ -0,0 +1,59 @@
...
+- [User interfaces using Material Design by Uplabs](https://www.uplabs.com/android).
```
Comment (moT01) [inline guide/spanish/android-development/material-design/index.md]:
```suggestion
- [User interfaces using Material Design by Uplabs](https://www.uplabs.com/android)
```

File: guide/spanish/android-development/material-design/index.md [hunk 20]
```diff
@@ -0,0 +1,59 @@
...
+
+# Cursos
+- [Material Design for Android Developers](https://www.udacity.com/course/material-design-for-android-developers--ud862).
```
Comment (moT01) [inline guide/spanish/android-development/material-design/index.md]:
```suggestion
- [Material Design for Android Developers](https://www.udacity.com/course/material-design-for-android-developers--ud862)
```

File: guide/spanish/android-development/material-design/index.md [hunk 21]
```diff
@@ -0,0 +1,59 @@
...
+- [Codelab](https://codelabs.developers.google.com/codelabs/material-design-style-sp/index.html#0).
```
Comment (moT01) [inline guide/spanish/android-development/material-design/index.md]:
```suggestion
- [Codelab](https://codelabs.developers.google.com/codelabs/material-design-style-sp/index.html#0)
```

File: guide/spanish/android-development/user-interface-desing/index.md [hunk 22]
```diff
@@ -0,0 +1,72 @@
+---
+title: User interface desing
```
Comment (moT01) [inline guide/spanish/android-development/user-interface-desing/index.md]:
```suggestion
title: User Interface Design
```

File: guide/spanish/android-development/user-interface-desing/index.md [hunk 23]
```diff
@@ -0,0 +1,72 @@
...
+setContentView(R.layout.main_activity);
+```
+El método setContentView despliega un layout, es decir una interfaz y la muestra en pantalla. Al desplegar un layout, se crea una instancia de cada elemento de diseño contenido en el archivo de layout, de acuerdo con la definición de sus atributos. Para especificar que layout se despliega pasamos al método el Id de recurso de dicho layout. 
+```
+Button button = (Button) findViewById(R.id.button);
+```
+Para conectar un elemento del archivo layout a la Activity utilizamos el método findViewById. Este método acepta el Id de recurso de un widget como argumento y devuelve un objeto View.
+
+# Más Información
+- [Layouts](https://developer.android.com/guide/topics/ui/declaring-layout?hl=en-419).
```
Comment (moT01) [inline guide/spanish/android-development/user-interface-desing/index.md]:
```suggestion
- [Layouts](https://developer.android.com/guide/topics/ui/declaring-layout?hl=en-419)
```

File: guide/spanish/android-development/user-interface-desing/index.md [hunk 24]
```diff
@@ -0,0 +1,72 @@
...
+- [Layout resource](https://developer.android.com/guide/topics/resources/layout-resource?hl=es-419).
```
Comment (moT01) [inline guide/spanish/android-development/user-interface-desing/index.md]:
```suggestion
- [Layout resource](https://developer.android.com/guide/topics/resources/layout-resource?hl=es-419)
```

File: guide/spanish/android-development/user-interface-desing/index.md [hunk 25]
```diff
@@ -0,0 +1,72 @@
...
+
+# Cursos
+- [Material Design for Android Developers](https://www.udacity.com/course/material-design-for-android-developers--ud862).
```
Comment (moT01) [inline guide/spanish/android-development/user-interface-desing/index.md]:
```suggestion
- [Material Design for Android Developers](https://www.udacity.com/course/material-design-for-android-developers--ud862)
```

File: guide/spanish/android-development/user-interface-desing/index.md [hunk 26]
```diff
@@ -0,0 +1,72 @@
...
+- [CodeLabs](https://codelabs.developers.google.com/codelabs/material-design-style-sp/index.html#0).
```
Comment (moT01) [inline guide/spanish/android-development/user-interface-desing/index.md]:
```suggestion
- [CodeLabs](https://codelabs.developers.google.com/codelabs/material-design-style-sp/index.html#0)
```

**Review by moT01 — CHANGES_REQUESTED** [review]
Thanks for the PR @FahedHermoza - for the most part, these looks real good. I made a number of suggestions - A lot of them are pretty minor, formatting issues. But a couple should get fixed up for sure. There's a typo in the title and the same typo in the folder name - as well as a few other typos in the english files where spanish was used. Also, we are trying to move away from using images in these articles, so it would be great if you could make these work without them. 

It looks like you spent some time on these and they look good, so it would be great if you could make these changes and we can probably get this merged. Thanks!

**Review by moT01 — APPROVED** [review]
Looks good @FahedHermoza - congratulations on your first contribution to freeCodeCamp! 🎉

**Pull request merged**